src/ontology/mondo-edit.obo

```diff
@@ -389059,6 +389078,15 @@ xref: OMIM:618056 {source="MONDO:equivalentTo"}
 xref: UMLS:CN252657 {source="MONDO:equivalentTo"}
 is_a: MONDO:0003847 ! Mendelian disease
 
+[Term]
+id: MONDO:0020842
+name: medullary carcinoma
```

Comment:
this is meant to be obsoleted? The label should say obsolete medullary carcinoma. I updated this, will commit in a minute.

Reply:
should we add a comment too with a reason for obsoletion?

Reply:
good catch!